<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>782</x>
     <y>110</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #95D1CC;
border-radius: 8px;
font-family: &quot;Trebuchet MS&quot;, Helvetica, sans-serif;
</string>
   </property>
   <property name="text">
    <string>Tambahkan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>450</y>
     <width>331</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #95D1CC;
border-radius: 8px;
font-family: &quot;Trebuchet MS&quot;, Helvetica, sans-serif;
font-size: 14px;</string>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hapusButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>490</y>
     <width>331</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #95D1CC;
border-radius: 8px;
font-family: &quot;Trebuchet MS&quot;, Helvetica, sans-serif;
font-size: 14px;
</string>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QListWidget" name="taskListWidget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>150</y>
     <width>331</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 6px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #95D1CC;
font: 24pt &quot;MS Shell Dlg 2&quot;;
color: #FAEDF0;
border-radius: 8px;
font-family: &quot;Trebuchet MS&quot;, Helvetica, sans-serif;</string>
   </property>
   <property name="text">
    <string>Planner Harian</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>471</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskLineEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 6px;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
